relay/config.go

```diff
@@ -166,6 +166,11 @@ func (prc *ProverConfig) getForkParameters() *lctypes.ForkParameters {
 					Epoch:   269568,
 					Spec:    &DenebSpec,
 				},
+				{
```

Comment:
According to the following documents, the version is 0x05000000 and the epoch is 364032:

* https://github.com/ethereum/EIPs/blob/ef468b1a357f16d3fdf7aaefe42d250fc513db06/EIPS/eip-7600.md#activation
* https://github.com/ethereum/consensus-specs/blob/8e92b52369fe66b01a8096dc2fb24835885dd494/specs/electra/fork.md#configuration

Here is a related PR: https://github.com/eth-clients/mainnet/pull/4

Reply:
spec is also available: https://github.com/ethereum/consensus-specs/blob/8e92b52369fe66b01a8096dc2fb24835885dd494/configs/mainnet.yaml#L54